kola/tests/bpf/bpf.go

```diff
@@ -40,8 +40,7 @@ func init() {
 		// required while SELinux policy is not correcly updated to support
 		// `bpf` and `perfmon` permission.
 		Flags: []register.Flag{register.NoEnableSelinux},
-		// exclude `arm64` while `quay.io/iovisor/bcc` does not have `arm64` support.
-		Architectures: []string{"amd64"},
+		Architectures: []string{"amd64", "arm64"},
```

Comment:
thanks, I think you can also remove this line as the default will be both

Reply:
ack, will update.